src/stylesheets/components/_card.scss

```diff
@@ -31,127 +56,278 @@
+  @include u-padding-bottom($theme-card-padding-perimeter);
+  @include u-padding-top($theme-card-padding-y / 2);
+  @include u-padding-x($theme-card-padding-perimeter);
 }
 
 .usa-card__footer .usa-button:only-of-type {
   @include u-margin-right(0);
 }
 
+.usa-card__header,
+.usa-card__body {
+  // strip spacing from first and last content
+  > :last-child {
+    @include u-padding-bottom(0);
+    @include u-margin-bottom(0);
+  }
+
+  > :first-child {
+    @include u-margin-top(0);
+    @include u-padding-top(0);
+  }
+
+  > :only-child {
+    @include u-margin-y(0);
+    @include u-padding-y(0);
+  }
+}
+
 // ---------------------------------
 // Variations
 // ---------------------------------
 
-// Raised
-// ---------------------------------
-.usa-card--raised {
-  @include u-shadow($theme-card-shadow);
+// Exdent
+.usa-card__header--exdent,
+.usa-card__media--exdent,
+.usa-card__footer--exdent {
+  @include u-margin-x(-$theme-card-border-width);
 }
 
-// Media
-// ---------------------------------
-.usa-card--media .usa-card__header {
-  @include u-padding-top($theme-card-padding);
-
-  &:last-child {
-    @include u-padding-bottom($theme-card-padding);
+.usa-card__header--exdent,
+.usa-card__footer--exdent {
+  > * {
+    @include u-padding-x($theme-card-border-width);
   }
 }
 
-.usa-card--media .usa-card__media {
+.usa-card__media--exdent {
   @include u-margin-top(-$theme-card-border-width);
-  @include u-margin-x(-$theme-card-border-width);
-  @include u-order("first");
-}
 
-.usa-card--media .usa-card__img {
-  @include u-radius-top($theme-card-border-radius);
-  overflow: hidden;
+  .usa-card__img {
+    @include u-radius-top($theme-card-border-radius);
+  }
 }
 
-
-// Flag
+// Header first
 // ---------------------------------
-.usa-card__inner {
-  @include at-media($theme-card-flag-min-width) {
-    @include grid-row;
-    @include u-padding-x(0);
-    @include u-width("full");
+.usa-card--header-first {
+  .usa-card__header {
+    @include card-inner-radius;
+    @include u-padding-bottom($theme-card-padding-y);
   }
-}
 
-.usa-card--flag .usa-card__media {
-  @include u-margin-top(-$theme-card-border-width);
-  @include u-margin-x(-$theme-card-border-width);
-  @include u-radius-top($theme-card-border-radius);
-  overflow: hidden;
+  .usa-card__header--exdent {
+    @include u-margin-top(-$theme-card-border-width);
+    @include u-radius-top($theme-card-border-radius);
+  }
 
-  @include at-media($theme-card-flag-min-width) {
-    @include grid-col(4);
-    @include u-margin-top(0);
-    @include u-margin-x(0);
-    @include u-padding-left($theme-card-padding);
-    @include u-padding-right(0);
-    @include u-padding-y($theme-card-padding);
-    @include u-radius-top(0);
+  .usa-card__media--inset {
+    @include u-padding-top(0);
   }
-}
 
-.usa-card--flag .usa-card__flag-content {
-  @include at-media($theme-card-flag-min-width) {
-    @include grid-col(8);
+  .usa-card__media {
+    @include u-order("initial");
```

Comment:
IE11 doesn't support `initial`. We should use `inherit` or any other value besides `unset/initial`.